src/main/modules/helmet/index.ts

```diff
@@ -7,7 +7,7 @@ export interface HelmetConfig {
 }
 
 const googleAnalyticsDomain = '*.google-analytics.com';
-const self = "'self'";
+const self = '\'self\'';
```

Comment:
don't be that guy

Reply:
Must have been my IDE 😅 I can revert this file